libs/utils/test.py

```diff
@@ -191,6 +191,12 @@ def get_end_times(self, experiment):
 
         return end_times
 
+    # Defining a default, empty runTest allows test classes to be conveniently
+    # instantiated. Since we run tests via nose, runTest isn't used. Instead all
```

Comment:
Are you saying that this method is required by the base class `unittest.TestCase`?
Perhaps I'm still under jet-la, but... I cannot really quite understand, from this comment, why we need to add this empty method?!